### PR TITLE
Build up the installation info in README

### PR DESCRIPTION
**Pull request by mwichmann**
Note: I didn't move virtualenv *first* as I'm finding discussion of virtualenv still makes many users surprisingly nervous.

Signed-off-by: Mats Wichmann <mats@linux.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -29,64 +31,138 @@ Contributing
...
 Contribute new tools by making a Pull Request on this repository.
-Please create a directory per contribution.
-(We may re-organize if it becomes obvious that doing so would 
+Please create a new directory per contribution.
+(We may re-organize if it becomes obvious that doing so would
 make it easier for users to find useful logic.)
 Please include a README file (README.rst or other form)
-describing any information about your tool, including the
-license it is released under.
+describing any information about your tool, *including the
```
Comment (bdbaddog) [inline README.rst:33]: Maybe "If at all possible please use the MIT license" ?

File: README.rst [hunk 2]
```diff
@@ -29,64 +31,138 @@ Contributing
...
+system-specific locations that are searched by default:
+a system-wide location, a user-account location, and inside
+a given project. These default locations are in the man page at
+`<https://scons.org/doc/production/HTML/scons-man.html#opt-site-dir>`_
+
+In addition to the defaults, the project-level site directory
+can be changed by supplying the ``--site-dir`` option,
+and SCons can also be given additional locations to look for
+tools using the ``toolpath`` parameter when creating a
+Construction Environment. See the manpage for details.
```
Comment (bdbaddog) [inline README.rst]: I'd leave this <<<<<=== text in. It's the simplest method to install a tool... and then include link to more info about the site dir? So they can get er done without having to click through to another doc..

Comment (mwichmann) [reply README.rst]: Sorry, leave what in?
Edited: oh, I think I get it. Thinking...

File: README.rst (hunk 1 above)
Comment (mwichmann) [reply README.rst:33]: Hmm, line 30 had a change I didn't intend, s/if/this/.

Comment (mwichmann) [reply README.rst:33]: > Maybe "If at all possible please use the MIT license" ?

made that change.

File: README.rst (hunk 2 above)
Comment (mwichmann) [reply README.rst]: Pushed a revised version which calls out `site_scons/site_tool`, before pointing off to the "for more info" stuff.

**Pull request merged**